app/controllers/ontologies_controller.rb

```diff
@@ -437,6 +437,61 @@ def metrics_evolution
     render partial: 'ontologies/sections/metadata/metrics_evolution_graph', locals: { data: data }
   end
 
+  def selector
```

Comment:
use a better name here, `selector` is too large, maybe replace it with `ontology_selector`